src/pages/en/guides/deploy/sst.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Deploy your Astro Site to AWS with SST
+description: How to deploy your Astro site to AWS with SST
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy an Astro site using [SST](https://sst.dev), an open-source framework for deploying fully serverless applications to AWS with SSG and SSR support.
+
+You can also use any additional SST constructs like Cron Jobs, Buckets, Queues, etc while maintaining type-safety.
+
+## Quickstart
+
+1. Create an astro project
+1. Run `npx create-sst@rc`
+1. It should detect that you are using Astro and ask you to confirm.
+1. Once you're ready for deployment you can run `npx sst deploy --stage=production`
+
+To use any [additional SST constructs](https://docs.sst.dev/), add them to `sst.config.ts`. 
+
+```ts {2} {4}
+app.stack(function Site(ctx) {
+  const bucket = new Bucket(ctx.stack, "public");
+  const site = new AstroSite(ctx.stack, "site", {
+    bind: [bucket],
+  });
+
+  ctx.stack.addOutputs({
+    url: site.url,
+  });
+});
+```
+
+And then access them in your astro code
+
+```ts
+import { Bucket } from "sst/node/bucket"
+console.log(Bucket.public.bucketName)
+```
```

Comment:
```````suggestion
And then access them in your astro code

```astro
---
import { Bucket } from "sst/node/bucket"
console.log(Bucket.public.bucketName)
---
```
``````
If this is a `.astro` file, you can use `astro` as the code language. Slight preference for throwing the code fences in there, too. But, otherwise, approved!